### PR TITLE
Update project-progression-policy.md

### PR DESCRIPTION
**Pull request by krtaylor**
Update project submission to include new LF project intake form

### CODE REVIEW

File: project-progression-policy.md [hunk 1]
```diff
@@ -44,8 +44,8 @@ and a vote by the Governing Board.
 #### Technical Review
 
 Projects are required to present their proposal at a TAC meeting.
-This proposal should cover all the items in the [Project Submission Template](https://github.com/confidential-computing/governance/blob/master/project-submission-template.md)
-and the [Technical Charter template](https://lists.confidentialcomputing.io/g/main/files/TAC/Project%20Submissions/Technical%20Charter%20%28custom+data%29%20--%20LF%20Projects,%20LLC%204-10-2019%20FINAL%20%2811%29.docx)).
+This proposal should cover all the items in the [Project Submission Template](https://github.com/confidential-computing/governance/blob/master/project-submission-template.md). The project should also complete the [LF Project Formation Intake Form](https://docs.google.com/forms/d/e/1FAIpQLSeO1bDGHUP-ZpCo1uynm94YOxZlek6RhCH7o3FnX1lZSXXfSQ/viewform), which will create the draft Technical Charter template, as well as other documents needed for approval. The project representative will receive a link to these drafts with a checklist of items to complete.
```
Comment (dcmiddle) [inline project-progression-policy.md:47]: Please line break markdown on sentences or at 100 character columns to facilitate diffs. (Though I recognize this file is inconsistent in that regard).
 ```suggestion
This proposal should cover all the items in the [Project Submission Template](https://github.com/confidential-computing/governance/blob/master/project-submission-template.md).
The project should also complete the [LF Project Formation Intake Form](https://docs.google.com/forms/d/e/1FAIpQLSeO1bDGHUP-ZpCo1uynm94YOxZlek6RhCH7o3FnX1lZSXXfSQ/viewform), which will create the draft Technical Charter template, as well as other documents needed for approval.
The project representative will receive a link to these drafts with a checklist of items to complete.
```

File: project-progression-policy.md [hunk 2]
```diff
@@ -72,8 +72,7 @@ the CCC requires that any trademarks be transferred to, or be available for
 use by, LF Projects, LLC.  Even if no trademarks exist, the current CCC
 policy is that projects be submitted to LF Projects, LLC.
 
-This submission entails sending the proposed Technical Charter (as reviewed
-by the TAC), and a signed [Project Contribution Agreement](https://lists.confidentialcomputing.io/g/main/files/TAC/Project%20Submissions/LF%20Projects%20--%20Form%20of%20Trademark%20and%20Account%20Assignment.docx).
+This submission entails sending the proposed Technical Charter (as reviewed by the TAC), and a signed Project Contribution Agreement that was generated by the LF Project Formation Intake Form listed above.
```
Comment (dcmiddle) [inline project-progression-policy.md:75]:
```suggestion
This submission entails sending the proposed Technical Charter (as reviewed by the TAC), 
and a signed Project Contribution Agreement that was generated by the LF Project Formation Intake Form listed above.
```

**Review by dcmiddle — COMMENTED** [review]
Looks good. A couple line break nits.

**Review by dcmiddle — APPROVED** [review]

File: project-progression-policy.md [hunk 3]
```diff
@@ -44,8 +44,10 @@ and a vote by the Governing Board.
 #### Technical Review
 
 Projects are required to present their proposal at a TAC meeting.
-This proposal should cover all the items in the [Project Submission Template](https://github.com/confidential-computing/governance/blob/master/project-submission-template.md)
-and the [Technical Charter template](https://lists.confidentialcomputing.io/g/main/files/TAC/Project%20Submissions/Technical%20Charter%20%28custom+data%29%20--%20LF%20Projects,%20LLC%204-10-2019%20FINAL%20%2811%29.docx)).
+This proposal should cover all the items in the [Project Submission Template](https://github.com/confidential-computing/governance/blob/master/project-submission-template.md).
+The project should also complete the [LF Project Formation Intake Form](https://docs.google.com/forms/d/e/1FAIpQLSeO1bDGHUP-ZpCo1uynm94YOxZlek6RhCH7o3FnX1lZSXXfSQ/viewform), which will create the draft Technical Charter template, as well as other documents needed for approval.
```
Comment (dthaler) [inline project-progression-policy.md:48]: I have comments on the form itself, but the text here is ok.

File: project-progression-policy.md [hunk 4]
```diff
@@ -72,8 +74,8 @@ the CCC requires that any trademarks be transferred to, or be available for
 use by, LF Projects, LLC.  Even if no trademarks exist, the current CCC
 policy is that projects be submitted to LF Projects, LLC.
 
-This submission entails sending the proposed Technical Charter (as reviewed
-by the TAC), and a signed [Project Contribution Agreement](https://lists.confidentialcomputing.io/g/main/files/TAC/Project%20Submissions/LF%20Projects%20--%20Form%20of%20Trademark%20and%20Account%20Assignment.docx).
+This submission entails sending the proposed Technical Charter (as reviewed by the TAC), 
+and a signed Project Contribution Agreement that was generated by the LF Project Formation Intake Form listed above.
```
Comment (dthaler) [inline project-progression-policy.md:78]: has Scott Nicolas verified that the generated agreement matches what is expected?

**Comment by krtaylor** [comment]
Closing due to intake form formatting, not ready to be integrated into the new project process

**Pull request closed without merging**